README.md

```diff
@@ -1,52 +1,58 @@
+tutorials or show-cases. These source files do not intend to substitute original their repositories.
 
-**Authors:**  [Carlos Mastalli](https://cmastalli.github.io/), Guilhem Saurel <br />
-**With additional support from the Gepetto team at LAAS-CNRS and The University of Edinburgh.**
+### :construction_worker: Authors
 
-[![pipeline status](https://gepgitlab.laas.fr/gepetto/example-robot-data/badges/master/build.svg)](https://gepgitlab.laas.fr/Gepetto/example-robot-data/commits/master)
+- [Carlos Mastalli](https://cmastalli.github.io/)
+- Guilhem Saurel
 
+With additional support from the Gepetto team at LAAS-CNRS :fr: and The University of Edinburgh :uk:
 
-## <img align="center" height="20" src="https://i.imgur.com/x1morBF.png"/> Installation
-You can install this package throught robotpkg. robotpkg is a package manager tailored for robotics softwares. It
-greatly simplifies the release of new versions along with the management of their dependencies. You just need to add
-the robotpkg apt repository to your sources.list and then use `sudo apt install robotpkg-example-robot-data` or `sudo
-apt install robotpkg-py27-example-robot-data` (or `py3X` for python 3.X, depending on your system) if you need the
-Python loaders.
+## :gear: Installation
 
-### Add robotpkg apt repository
-If you have never added robotpkg as a softwares repository, please follow first the instructions from 1 to 3. Otherwise, go directly to instruction 4. Those instructions are similar to the installation procedures presented in [http://robotpkg.openrobots.org/debian.html](http://robotpkg.openrobots.org/debian.html).
+### :penguin: From Debian / Ubuntu packages, with [robotpkg](http://robotpkg.openrobots.org)
 
-1. Add robotpkg as source repository to apt:
+1. If you have never added robotpkg's software repository, [do it now](http://robotpkg.openrobots.org/debian.html):
+   ```bash
+   sudo tee /etc/apt/sources.list.d/robotpkg.list <<EOF
+   deb [arch=amd64] http://robotpkg.openrobots.org/packages/debian/pub $(lsb_release -sc) robotpkg
+   EOF
 
-		sudo tee /etc/apt/sources.list.d/robotpkg.list <<EOF
-		deb [arch=amd64] http://robotpkg.openrobots.org/wip/packages/debian/pub $(lsb_release -sc) robotpkg
-		deb [arch=amd64] http://robotpkg.openrobots.org/packages/debian/pub $(lsb_release -sc) robotpkg
-		EOF
+   curl http://robotpkg.openrobots.org/packages/debian/robotpkg.key | sudo apt-key add -
+   sudo apt update
+   ```
 
-2. Register the authentication certificate of robotpkg:
+2. installation of example-robot-data and its python utils:
+   ```bash
+   sudo apt install robotpkg-py3\*-example-robot-data
+   ```
 
-		curl http://robotpkg.openrobots.org/packages/debian/robotpkg.key | sudo apt-key add -
+3. If you install this in a non-standard directory and **without catkin**, you will need to set `ROS_PACKAGE_PATH` to
+   your `$CMAKE_INSTALL_PREFIX/share` (`/usr/local/share` by default)
```

Comment:
Does this mean that robotpkg does *not* install to `/opt/openrobots/share` but to `/usr/local/share`?

Reply:
`/usr/local` is the default `CMAKE_INSTALL_PREFIX. But you are right, we should talk about robotpkg.

Reply:
I think we should also add as prominently as a second installation option that simply cloning this into a catkin workspace works just fine, no further configuration needed.

The /usr/local should be option 3 - pure CMake locally.

Reply:
I actually wrote the "install from source" part as you described, and it took a lot of place for only standard boring stuff, so I removed it. What do you think of the current state ? We could add a catkin part, especially if this is really short :)